src/qtism/runtime/tests/Utils.php

```diff
@@ -125,4 +139,10 @@ public static function isResponseValid(QtiDatatype $response = null, ResponseVal
 
         return true;
     }
+
+    private static function isSingleMatchGroup(string $patternMask): bool
+    {
+        $closeBracketPosition = strpos($patternMask, self::CLOSE_MATCH_GROUP_TOKEN);
```

Comment:
If it's not too much of a trouble, could you please add a few test cases that would check this condition?